frontend/src/bundles/common/helpers/date-converter/date-converter.ts

```diff
@@ -0,0 +1,9 @@
+import { DateObject } from 'react-multi-date-picker';
+
+const dateConverter = (date: string): string => {
+    const convertedDate = new DateObject(date);
+
+    return convertedDate.format('dddd, MMMM DD | HH:mm A');
+};
+
```

Comment:
rename to formatDate or something similar, function name should start with noun. Also we already have similar helper, maybe it's worth to create formatDate(date, format) where format will be value from enum or so.